launchdarkly/config.go

```diff
@@ -8,7 +8,7 @@ import (
 	ldapi "github.com/launchdarkly/api-client-go"
 )
 
-const Version = "0.0.1"
+const Version = "0.0.2"
```

Comment:
Let's bump this to 1.0.0. The provider is no longer in beta -- we fully support it and let's indicate as much with our version string.

From https://semver.org/#spec-item-4:
> Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.